packages/ruleset/src/collections/index.js

```diff
@@ -1,6 +1,9 @@
 // A group of predefined "collections" of OpenAPI locations to validate,
 // to be re-used by multiple rules.
 
+// A collection of locations of top-level response schemas
+const responseSchemas = ['$.paths[*][*][responses][*].content[*].schema'];
```

Comment:
Should we restrict this to just the keys under which we'd expect to find an operation, similar to `operations` below?
If yes, we probably need to update `unresolvedResponseSchemas` below as well :)

Reply:
Opened an issue: https://github.com/IBM/openapi-validator/issues/438